web-components/src/components/organisms/PostFiles/PostFiles.Public.tsx

```diff
@@ -85,35 +85,20 @@ const PostFilesPublic = ({
     })),
   };
 
-  const onLoad = (): void => {
-    if (locations) {
-      const [minLng, minLat, maxLng, maxLat] = bbox(locations);
-      mapRef.current?.fitBounds(
```

Comment:
The issue is that `fitBounds` receives the same coordinates in both `[minLng, minLat]` and `[maxLng, maxLat]` from `bbox(locations)` because `bbox` only receives one coordinate from `locations`. The result is the map setting the box boundaries to a single point and zooming in to close to it, overwriting the `initialViewState.zoom`. 

If we are going to receive only a single coordinate, it's better to only use `initialViewState`. However, if there are scenarios where we expect multiple coordinates , please let me know and I'll add support for that case.

Reply:
The downside of removing the `onLoad` `fitBounds` is that we could have multiple files with different locations, and only the first point from the first location might be in the map viewport, right? 

Would it make sense to keep the `onLoad` `fitBounds` solution, and just pass the `maxZoom` option to prevent it from getting too zoomed in? 
https://docs.mapbox.com/mapbox-gl-js/api/map/#map#fitbounds

Reply:
Thanks for clarifying this. I have added back the `fitBounds` to handle situations where we get more than one coordinate. In the end there's no need for controlling the zoom as `fitBounds` automatically sets the right zoom to fit all the markers.